src/field/vecops.cxx

```diff
@@ -239,12 +241,13 @@ const Vector2D Curl(const Vector2D &v, CELL_LOC outloc) {
 
   TRACE("Curl( Vector2D )");
 
-  Mesh *localmesh = v.x.getMesh();
-
   if (outloc == CELL_DEFAULT) {
     outloc = v.getLocation();
```

Comment:
Not sure if this comment is to contrast the earlier ones, but just in case you didn't spot it the subsequent line asserts that the location isn't vshift, so will indeed fail.